sigar_windows_test.go

```diff
@@ -117,6 +117,19 @@ var _ = Describe("SigarWindows", func() {
 		})
 	})
 
+	Describe("NetProtoStats", func() {
+		It("gets IPv4", func() {
+			stats := sigar.NetProtoV4Stats{}
+			err := stats.Get()
+			Ω(err).ShouldNot(HaveOccurred())
+		})
+		It("gets IPv6", func() {
+			stats := sigar.NetProtoV6Stats{}
+			err := stats.Get()
+			Ω(err).ShouldNot(HaveOccurred())
+		})
+	})
```

Comment:
The values on windows are tough to assert against because they're arbitrary from the point of view of the test. Some of them could be asserted nonzero, but I'm not 100% sure that would pass in all environments. Unfortunately its not like the linux side where we can just parse a sample file. If we restructured though, we could slap an interface in there and provide a test mock. I put a JIRA on the back burner for this.